html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+<h1>Coding Form!</h1>
+
+<form action="">
+  <label for="email">Email <br />
+  <input type="text" id="email" name="email">
+  </label>
+  <div>
+  <label for="whyCode">Why is coding cool? <br />
+    <select name="codingCool" id="whyCode">
+      <option value="selectOne">Please select one</option>
+      <option value="interesting">Becuase its interesting</option>
+      <option value="fun">Becuase its fun</option>
+      <option value="powerful">Becuase its powerful</option>
+    </select>
+  </label>
+  </div>
+  <div>
+  <label for="language">Favorite Language (Choose one) <br />
+    <label for="html">
+      <input type="radio" id="html" name="html" value="html">HTML
+    </label>
+    <label for="css">
+      <input type="radio" id="css" name="css" value="css">CSS
+    </label>
+    <label for="javascript">
+      <input type="radio" id="javascript" name="JavaScript" value="JavaScript">JavaScript
+    </label>
```

Comment:
For the radio buttons to work properly, all the related radio buttons should have the same `name` attribute value set. This will make it so only one radio button can be selected at a time.